src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -358,127 +212,15 @@ export const View = (props: Props) => {
-                        "DisableOneRepScans",
-                      ) &&
-                      (typeof props.totalNumberOfPerformedScans ===
-                        "undefined" ||
-                        props.totalNumberOfPerformedScans <
-                          CONST_ONEREP_MAX_SCANS_THRESHOLD) ? (
-                        <Link
-                          ref={waitlistTriggerRef}
-                          href="/user/welcome/free-scan?referrer=dashboard"
-                          onClick={() => {
-                            recordTelemetry("link", "click", {
-                              link_id: "exposures_all_fixed_free_scan",
-                            });
-                          }}
-                        />
-                      ) : (
-                        <Button
-                          variant="link"
-                          buttonRef={waitlistTriggerRef}
-                          {...overlayTrigger.triggerProps}
-                        />
-                      ),
-                  },
-                },
-              )
-            : l10n.getFragment("dashboard-exposures-all-fixed-free-scan", {
-                vars: {
-                  data_broker_total_num: CONST_ONEREP_DATA_BROKER_COUNT,
-                },
-                elems: {
-                  a:
-                    !props.enabledFeatureFlags.includes("DisableOneRepScans") &&
-                    (typeof props.totalNumberOfPerformedScans === "undefined" ||
-                      props.totalNumberOfPerformedScans <
-                        CONST_ONEREP_MAX_SCANS_THRESHOLD) ? (
-                      <Link
-                        ref={waitlistTriggerRef}
-                        href="/user/welcome/free-scan?referrer=dashboard"
-                        onClick={() => {
-                          recordTelemetry("link", "click", {
-                            link_id: "exposures_all_fixed_free_scan",
-                          });
-                        }}
-                      />
-                    ) : (
-                      <Button
-                        variant="link"
-                        buttonRef={waitlistTriggerRef}
-                        {...overlayTrigger.triggerProps}
-                      />
-                    ),
-                },
-              })
-          /* c8 ignore stop */
-        }
-      </p>
-    </>
-  );
-
   const getZeroStateIndicator = () => {
-    if (initialScanInProgress) {
-      return (
-        <>
-          <Image src={ScanProgressIllustration} alt="" />
-          <strong>
-            {l10n.getString("dashboard-exposures-scan-progress-label")}
-          </strong>
-        </>
-      );
-    }
-
-    if (!hasUnresolvedExposures && hasFixedExposures) {
-      return (
-        <>
-          <Image src={AllFixedIllustration} alt="" />
-          <strong>
-            {l10n.getString("dashboard-exposures-all-fixed-label")}
-          </strong>
-          {freeScanCta}
-        </>
```

Comment:
this seems still relevant? if I'm reading it correctly it should also be shown for if you fixed all breaches? or is this broker-specific

Reply:
Good eye - I saw the string is localized so I added it back in https://github.com/mozilla/blurts-server/pull/6314/commits/718c01be9b20b8ca69396039cbbd20858af7aa53.